### PR TITLE
BNGP-4702 credits-purchase : mini cya page

### PR DESCRIPTION
**Pull request by jd-barnard**

### CODE REVIEW

File: packages/webapp/src/routes/credits-purchase/cdd-cya.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  const nationality = session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_NATIONALITY)
+  const nationalityHtml = nationality ? Object.values(nationality).filter(n => n !== '').join('<br/>') : ''
+
+  return {
+    dueDiligence: {
+      individualOrOrg: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_USER_TYPE),
+      individualOrOrgUrl: creditsPurchaseConstants.routes.CREDITS_PURCHASE_INDIVIDUAL_OR_ORG,
+      checkDefraAccount: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_DEFRA_ACCOUNT_DETAILS_CONFIRMED) && 'Confirmed',
+      checkDefraAccountUrl: creditsPurchaseConstants.routes.CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS,
+      middleName: session.get(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_MIDDLE_NAME)?.middleName,
```
Comment (KiranVarma02) [inline packages/webapp/src/routes/credits-purchase/cdd-cya.js:14]: is this right? CREDITS_PURCHASE_MIDDLE_NAME is an object? not a value?

File: packages/webapp/src/views/credits-purchase/cdd-cya.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            text: "Save and continue",
+            attributes: {
+            id: "continue",
+            "data-testid": "continue"
+            }
+            }) }}
+        </div>
+    </div>
+</div>
+{% endblock %}
```
Comment (KiranVarma02) [inline packages/webapp/src/views/credits-purchase/cdd-cya.html:80]: something is not right with your editor and formatting :)

File: packages/webapp/src/routes/credits-purchase/cdd-cya.js (hunk 1 above)
Comment (jd-barnard) [reply packages/webapp/src/routes/credits-purchase/cdd-cya.js:14]: Yes it is - https://github.com/DEFRA/biodiversity-net-gain-service/blob/e7725f2e04239df262fb5e742304ae02122c85cc/packages/webapp/src/routes/credits-purchase/middle-name.js#L32

File: packages/webapp/src/views/credits-purchase/cdd-cya.html (hunk 2 above)
Comment (jd-barnard) [reply packages/webapp/src/views/credits-purchase/cdd-cya.html:80]: I've associated twig with all html files and I think it looks better now...

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**